cmd/nerdctl/events.go

```diff
@@ -35,6 +38,19 @@ var eventsCommand = &cli.Command{
 	Usage:       "Get real time events from the server",
 	Description: "NOTE: The output format is not compatible with Docker.",
 	Action:      eventsAction,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "format",
+			Usage: "Format the output using the given Go template, e.g, '{{json .}}'",
+		},
+	},
+}
+
+type Event struct {
+	Timestamp time.Time
+	Namespace string
+	Topic     string
+	Out       string
```

Comment:
if use  Out instead of Event it will be that,
type Out struct {
	Timestamp time.Time
	Namespace string
	Topic     string
	Out       string
}   
which will have double out,not beautiful.

Reply:
I’m suggesting to use “Event”, not “Out”

Reply:
ping, LGTM after s/Out/Event/. Thanks.

Reply:
change  from 
type Event struct {
	Timestamp time.Time
	Namespace string
	Topic     string
	Out       string
}
to 
type Out struct {
	Timestamp time.Time
	Namespace string
	Topic     string
	Event       string
}
?

Reply:
Yes